package.json

```diff
@@ -13,6 +13,9 @@
     "build.prod": "gulp build.prod --color",
     "build.test": "gulp build.test --color",
     "build.test.watch": "gulp build.test.watch --color",
+    "generate.manifest.dev": "gulp generate.manifest.dev --color",
+    "generate.manifest.prod": "gulp generate.manifest.prod --color",
+    "docs": "npm run gulp -- build.docs --color && npm run gulp -- serve.docs --color",
```

Comment:
`docs` can be removed, this has been refactored by @d3viant0ne and is no longer there.